server/gorm/client.go

```diff
@@ -258,10 +258,9 @@ func (c *Client) Delete(ctx context.Context, k storage.Key) error {
 func (c *Client) Run(ctx context.Context, q storage.Query) storage.Iterator {
 	mylock()
 	defer myunlock()
-	limit := q.(*Query).Limit
 	cursor := q.(*Query).Cursor
 
-	op := c.db.Limit(limit).Where("key > ?", cursor)
+	op := c.db.Where("key > ?", cursor)
```

Comment:
I notice this gets everything - you probably already thought about this, but we should keep an eye on performance for large collections

Reply:
Yeah we might want to optimize this later by reintroducing a limit and having filtered listing construct a response from multiple reads. This should be fine for even a few thousand specs though if I had to guess, due to their small memory footprint (contents aren't loaded). Load testing would reveal the true limits of this approach.